Exercise_4/docs/part-3.md

```diff
@@ -1 +1,204 @@
+![uscore_medications](img/uscoremedications1.png)
+
+We can see details of the elements of the medication profile: [http://build.fhir.org/ig/HL7/US-Core/StructureDefinition-us-core-medication.html](http://build.fhir.org/ig/HL7/US-Core/StructureDefinition-us-core-medication.html){target=_blank}
+
+??? question "Knowledge Check: Which element(s) of this profile must be supported in order to be US Core Compliant?"
+    `Medication.code` element
+
+Going back to our **smoking example** take a look here: [http://hl7.org/fhir/us/core/STU5/StructureDefinition-us-core-smokingstatus.html#mandatory-and-must-support-data-elements](http://hl7.org/fhir/us/core/STU5/StructureDefinition-us-core-smokingstatus.html#mandatory-and-must-support-data-elements){target=_blank}
+
+![Smoking core profile part 1](img/smokingcore1.png)
+![Smoking core profile part 2](img/smokingcore2.png)
+
+
+We can see here a list of required elements this profile must support.
+
+??? question "Knowledge Check: How many elements are required and what are they?"
+    7 required elements. They include a `status`, a category slice, and then a specific category code of `social-history`, a code for smoking observation, a patient, when the observation occurred, a result value code for smoking status
+
+??? question "Knowledge Check: Where would I find the reference ID of the patient?"
+    `Subject code`
+
+Let’s take a look at the raw unformatted **JSON code** for the resource:
+
+??? example "Sample JSON code of the Smoking Resource"
+    ```JSON
+    {
+    "resourceType": "Observation",
+    "id": "056f4fe8-cd0a-db01-9d8b-0b3bf098368c",
+    "meta": {
+        "profile": [ "http://hl7.org/fhir/us/core/StructureDefinition/us-core-smokingstatus" ]
+    },
+    "status": "final",
+    "category": [{
+        "coding": [{
+            "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+            "code": "social-history",
+            "display": "social-history"
+        }]
+    }],
+    "code": {
+        "coding": [{
+        "system": "http://loinc.org",
+        "code": "72166-2",
+        "display": "Tobacco smoking status NHIS"
+        }],
+        "text": "Tobacco smoking status NHIS"
+    },
+    "subject": { "reference": "urn:uuid:c37a6b5c-a13c-835d-bfe2-e215df91cb3c" },
+    "encounter": { "reference": "urn:uuid:0f7f8554-0896-65dd-7bed-fe85762791dd" },
+    "effectiveDateTime": "2012-07-25T10:07:28-04:00",
+    "issued": "2012-07-25T10:07:28.029-04:00",
+    "valueCodeableConcept": {
+        "coding": [{
+        "system": "http://snomed.info/sct",
+        "code": "266919005",
+        "display": "Never smoker"
+        }],
+        "text": "Never smoker"
+    }
+    }
+    ```
+Check out this patient profile here: HL7.FHIR.US.CORE\Some Day Smoker Example - JSON Representation [http://hl7.org/fhir/us/core/STU5/Observation-some-day-smoker.json.html](http://hl7.org/fhir/us/core/STU5/Observation-some-day-smoker.json.html){target=_blank}
```

Comment:
```suggestion
See another example of a Smoking Status Observation here: [http://hl7.org/fhir/us/core/STU5/Observation-some-day-smoker.json.html](http://hl7.org/fhir/us/core/STU5/Observation-some-day-smoker.json.html){target=_blank}
```

It's confusing to call this a patient profile since that has a real and different meaning in FHIR.

Reply:
Should this be linking to the same example as above and just say `Source: `